plugins/community/roadrunner-server-php-grpc/v4.2.0/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# syntax=docker/dockerfile:1.4
+FROM golang:1.20.6-bullseye AS build
+RUN --mount=type=cache,target=/go/pkg/mod \
+    git clone --depth=1 --branch=v4.2.0 https://github.com/roadrunner-server/grpc.git \
```

Comment:
Can't use `go install` here because the module has a `v2` w/ a v4.2.0 tag.

Reply:
Yeah, I'll update the module version, sure.

Reply:
Thanks @rustatian - not a huge deal but that would bring this more in line with other Go based protoc plugins we build here. I mainly left it as a note for our team.

Reply:
Done: https://github.com/roadrunner-server/grpc/releases/tag/v4.3.0

Reply:
To use go install - i.e.:

```
$ go install -ldflags "-s -w" -trimpath github.com/roadrunner-server/grpc/protoc_plugins/v4/protoc-gen-php-grpc@v4.3.0
```

a module needs to have its own tag - i.e. `protoc_plugins/v4.3.0` - see https://github.com/grpc/grpc-go/tree/cmd/protoc-gen-go-grpc/v1.3.0/cmd/protoc-gen-go-grpc as an example. Alternatively, the CLI would need to move to the same module as the root of the project.

Reply:
See https://go.dev/doc/modules/managing-source#multiple-module-source for more info.

Reply:
Oh, yeah, got u, forget about `go.work`, gimme a second

Reply:
Done, now should work 😃